app/views/users/edit.html.erb

```diff
@@ -1 +1,3 @@
-<%= render "edit_form", user: @user, weekdays: @weekdays, sports: @sports %>
+<div class="container">
+  <%= render "edit_form", user: @user, weekdays: @weekdays, sports: @sports %>
```

Comment:
solved the conflict by adding the weekdays and sports to the render line, which were changes I made in an earlier commit to add features to the edit page